docs/team/dandaandaaaaaan.md

```diff
@@ -2,39 +2,47 @@
+ContaX is an integrated solution designed for busy professionals who work with many clients and frequently interact with other people. It assists those who have packed schedules in efficiently scheduling appointments, while also being able to respond to volatile changing schedules.
 
-ContacX will allow for efficient management of a large list of contacts, allowing the mental workload of remembering people and related scheduled tasks to be offloaded. It will also provide an efficient way of efficiently organizing resources, while having minimal downtime with seamless transition from existing solutions.
+ContaX allows efficient management of a large list of contacts together with a schedule, providing an integrated solution for tracking work-related information. Built with efficiency in mind, ContaX promises a seamless transition from existing solutions for minimal downtime.
 
 Given below are my contributions to the project.
 
 * **New Feature**: Added the import and export features for `.csv`
     * What it does: Allows the user to import contacts from a Microsoft Excel compatible format, as well as export it back into the same format. Import CSV allows for customising of column numbers for each field.
     * Justification: As our target audience is professionals, it is very common that they will have contacts in Excel in some form. By designing an Import CSV command with flexibility of column numbers, professionals can easily migrate their existing contacts with minimal modifications.
+* **New Feature**: Internal parser to style text components with a Markdown-like syntax
+    * What it does: Allows for dynamically generated styled text, through Bold/Italic/Monospaced text, mainly used in command usage messages. e.g. a String of `**Bold**` will display as **Bold**, similar to in Markdown. 
+    * Justification: In order to allow for developers to easily create styled text as feedback to users. Some text components can get very long, especially command usage messages. Through the use of this parser, information can be presented to the users in a more organised manner, making it easier to read.
 
 * **Code contributed**:
+    * 2000+ LoC contributed to the project. See [Reposense](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=dandaandaaaaaan) for more details.
     * Functionality, documentation and tests for both `Import CSV` and `Export CSV` functions
       * Includes components in `Storage`, `Logic`, `Model` etc.
-
+    * Functionality, documentation and tests of Markdown-like test styler
+      * Includes components mainly in `UI`
+      
 * **Project management**:
-    * Code reviews, workflow establishment
-    * Creation of releases, ensuring passing CI
+    * Established the overall team workflow.
+    * Ensuring code quality through reviews.
+    * Creation of releases, ensuring CI passing.
```

Comment:
Consider the following phrasing.
```suggestion
    * Creating and Managing the lifecycle of releases, including version number maintainence in code.
    * Ensuring that CI failures are resolved in a timely manner.
```

Reply:
thx for suggestion, implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/316/commits/fbd807ab751c170f9493d3a19fc13c8d319cd5f1